### ISSUE TITLE
HTML Alt text

### ISSUE THREAD

**Issue by mHammy** [mHammy/c1-horiseon-refactor#2]
The index.html file is missing accessibility standards

**Comment by mHammy** [comment]
Updated all of the alt text to roughly describe the images/icons on the page for accessibility.
